src/components/PulseWidgets.tsx

```diff
@@ -9,7 +9,8 @@ import { useWidgetRenderer, StylingProps } from "./renderer"
 export type PulseWidgetProps = SsoUrlProps &
   StylingProps &
   WidgetConfigurationProps &
-  PulsePostMessageCallbackProps<string>
+  PulsePostMessageCallbackProps<string> &
+  JSX.IntrinsicAttributes
```

Comment:
Needed after migrating from React v17 to v18 (SDK uses 17, example app uses 18). It's possible we'll be able to revert this change after upgrading the SDK.